File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int n, i, j;

    //obtain the number of layers in the pyramid
    do
    {
        n = get_int("Height: ");
    }
    while (n < 0 || n > 23);

    //print the pyramid to screen
    for (i = 0; i < n; i++)
    {
        for (j = 0; j < n + 3 + i; j++)
        {
            if (j < n - i - 1 || j == n || j == n + 1) //this statement judges when there should be a space present
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: pset6/mario/mario.py
# this program accepts user inputed layers and prints the correspondign pyramid to the screen

from cs50 import get_int

# prompt user for height from 1 - 23
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# print pyramid to screen

for i in range(n):
    for j in range(n + 3 + i):
        if (j < n - i - 1) or (j == n + 1) or (j == n):
            print(" ", end="")
        else:
            print("#", end="")
    print()
